content/parts/en/contributor-form.mdx

```diff
@@ -0,0 +1,15 @@
+## Sign your contribution
+
+Open Terms Archive is an open source initiative that relies on contributors just like you 🤗
+
+By default, your contributions will be anonymous. Signing your contributions enables you to be credited publicly for them and to be notified when they are integrated or discussed.
+
+<ContributorFormFields />
+
+- While not readable anywhere on the web, this email address will be [publicly](https://stackoverflow.com/a/897604/594053) and irreversibly associated with your contribution.
+- We will use this address only if it is necessary to contact you for integrating, crediting or otherwise handling this specific contribution.
+- If you have an account on [GitHub](https://github.com), using an address linked to it will enable mentioning your account.
+
+<small>
+  If you only want to contribute anonymously, you can leave the default email. You can always join the discussion later, but maintainers will not be able to proactively contact you if they have questions.
```

Comment:
> You can always join the discussion later, but maintainers will not be able to proactively contact you if they have questions.

I assume this is mainly referring to joining a PR discussion, and maintainers contacting related to the PR before merging?

With the email use disclosure also listing “otherwise handling this specific contribution” I was also thinking of if there is relicensing and other copyright related requests/communication. Is that also considered a possibility, and if not, what does other handling refer to/enclose?